### PR TITLE
Faculty Directory migration

### PR DESCRIPTION
**Pull request by mandythomas**
moved all js, css and images for faculty directory templates (4 types
including Law specific one) into github so not local to Cascade CMS anymore. More like Modular templates

### CODE REVIEW

File: app/assets/stylesheets/layout_old/faculty/main-faculty-search-results.scss [hunk 1]
```diff
@@ -1,47 +1,62 @@
...
+		border-bottom: none;
+		margin-left: 5px;
+		padding:10px;
+		max-width: 120px;
+			//padding: 5px 15px 5px 15px;
+			//background-color: #8f0c30;
+			//color: $white;
+			// font-family: "Open Sans";
+			//font-size: 11px;
+			//font-weight: 800;
```
Comment (tatwell) [inline app/assets/stylesheets/layout_old/faculty/main-faculty-search-results.scss]: Dead code.

**Comment by tatwell** [comment]
I haven't tried to divine what the javascript code here is trying to do. It looks like the changes were aimed mainly at scoping it more narrowly.

My impression is that whatever the code is trying to do, it's trying to do too much. But so long as it keeps doing what it was doing, I'm good with it.

**Comment by mirabelrice** [comment]
Some of the javascript is a little hard to follow, but it works for now 👍

File: app/assets/stylesheets/layout_old/faculty/school-dept-faculty-search-results.scss [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+				border-left: 1px solid $black;
+				text-decoration: underline;
+				cursor: hand;
+				cursor: pointer;
+      		}
+		  @include bp(m) {
+			float: right;
+		  }
+    	} //end pagingNavigation
+  	} // end pagingInfo
```
Comment (tatwell) [inline app/assets/stylesheets/layout_old/faculty/school-dept-faculty-search-results.scss]: Nesting in this block is off.

**Comment by tatwell** [comment]
👍  Still some minor formatting issues. But those can be cleaned up at a later date.
